tests/test_backend.py

```diff
@@ -425,8 +424,10 @@ def test_file(tmpdir, src_path, dst_path, version, backend):
 )
 def test_ls(tmpdir, backend):
 
-    assert backend.ls() == []
```

Comment:
This is related to #86. In fact, so far the repositories we use in our tests are only created in the moment we put the first file on the backend. The reason this didn't raise an error before was that the backend was returning an empty list when something went wrong, which was actually hiding the original problem of a non-existing repository.